source/linux/Foundational_Components/U-Boot/UG-General-Info.rst

```diff
@@ -211,10 +211,19 @@ Build U-Boot
 
    Several prebuilt images are required from the TI Processor SDK for building U-Boot on K3 based platforms.
 
-   - TF-A (BL31): Refer to :ref:`foundational-components-atf` for more information
-   - OP-TEE (TEE): Refer to :ref:`foundational-components-optee` for more information
-   - ti-linux-firmware (BINMAN_INDIRS): Prebuilt binaries for DM and SYSFW available `here
-     <https://git.ti.com/cgit/processor-firmware/ti-linux-firmware/log/?h=ti-linux-firmware>`_.
+   .. ifconfig:: CONFIG_part_variant not in ('AM62LX')
+
+      - TF-A (BL31): Refer to :ref:`foundational-components-atf` for more information
+      - OP-TEE (TEE): Refer to :ref:`foundational-components-optee` for more information
+      - ti-linux-firmware (BINMAN_INDIRS): Prebuilt binaries for DM and SYSFW available `here
+        <https://git.ti.com/cgit/processor-firmware/ti-linux-firmware/log/?h=ti-linux-firmware>`_.
```

Comment:
Two `_` for an anonymous link

Reply:
oops. yeah I should have fixed it while I was there